### DISCUSSION TITLE
Font Changes?

### DISCUSSION THREAD

**Discussion by adeshpande03** [DenverCoder1/github-readme-streak-stats#709] [Q&A]
How would I change the font to a google font?

**Comment by adeshpande03** [comment]
Ah ok I figured this out. Followed this guide from ChatGPT, and find and replaced all segoe ui with my font in card.php. 

![image_2024-08-07_211459888](https://github.com/user-attachments/assets/49a11bb5-70e9-446a-aeef-6c77b90c079d)

**Reply by Cfomodz** [reply]
Can be marked as answered :)
